quodlibet/po/da.po

```diff
@@ -4641,7 +4641,7 @@ msgid "Tag expression"
 msgstr "Tag-udtryk"
 
 #: ../quodlibet/qltk/data_editors.py:373
-msgid "Tag expression e.g. people:real or ~album~year."
+msgid "Tag expression e.g. people:real or ~album~year"
 msgstr "Tag-udtryk. F.eks. people:real eller ~album~year."
```

Comment:
I think the period was missed

Reply:
It looks like two sentences in the translation, so the translator might want to retain the ending punctuation.

Reply:
But given all the uncertainties that arise, it might indeed be better in the future to not touch the PO files, and let the translators decide what to do; see https://github.com/quodlibet/quodlibet/pull/3218#discussion_r302978599.

Reply:
Ah, I see, the period after udtryk is surprising. 

> touch the PO files, and let the translators decide what to do; see #3218 (comment).

True. I wasn't aware about the fuzzy matching thing. That said, it still requires extra work from the translators and given that that work might only happen very rarely, I think updating in the very obvious cases still makes sense. But yeah, that's not the case here.